modules/core/shared/src/main/scala/lucuma/core/model/GuideConfig.scala

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats.Eq
+import cats.derived.*
+import cats.syntax.all.*
+import coulomb.syntax.*
+import coulomb.units.accepted.Millimeter
+import io.circe.Decoder
+import io.circe.DecodingFailure
+import io.circe.Encoder
+import io.circe.Json
+import io.circe.syntax.*
+import lucuma.core.enums.ComaOption
+import lucuma.core.enums.Cwfs1Usage
+import lucuma.core.enums.Cwfs2Usage
+import lucuma.core.enums.Cwfs3Usage
+import lucuma.core.enums.GuideProbe
+import lucuma.core.enums.M1Source
+import lucuma.core.enums.MountGuideOption
+import lucuma.core.enums.OIUsage
+import lucuma.core.enums.Odgw1Usage
+import lucuma.core.enums.Odgw2Usage
+import lucuma.core.enums.Odgw3Usage
+import lucuma.core.enums.Odgw4Usage
+import lucuma.core.enums.P1Usage
+import lucuma.core.enums.TipTiltSource
+import lucuma.core.model.AltairConfig.*
+import lucuma.core.model.GemsConfig.*
+import lucuma.core.model.M1GuideConfig.*
+import lucuma.core.model.M2GuideConfig.*
+import lucuma.core.model.TelescopeGuideConfig.given
+import monocle.Focus
+import monocle.Lens
+import monocle.Optional
+import monocle.std.either
+import monocle.std.option
+
+case class GuideConfig(
+  tcsGuide:      TelescopeGuideConfig,
+  gaosGuide:     Option[Either[AltairConfig, GemsConfig]],
+  gemsSkyPaused: Boolean
```

Comment:
I know nothing whatsoever about this, but I wonder why `gemsSkyPaused` isn't just part of `GemsConfig`.

Reply:
Ooh, someone is exposing my dirty secrets. `gemsSkyPaused` is just an internal flag used when dealing with GeMS, it is not related to the GeMS configuration applied in TCC. I just needed a place to store it, and `GuideConfig`  seemed convenient at the time.

Reply:
Seemed a bit odd that it'd be there whether or not working with GeMS.